src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -239,6 +257,8 @@ protected function rollbackMigrations(array $migrations, $paths, array $options)
 
         $this->requireFiles($files = $this->getMigrationFiles($paths));
 
+        $this->fireMigrationEvent('beforeAll:down');
```

Comment:
Think you forgot this one?

Reply:
Damnit. Its on my imac at home, so i'll be able to fix it this evening.